src/components/BookList.js

```diff
@@ -17,22 +20,41 @@ const BookList = ({ books }) => {
   };
 
   return (
-    <ul>
+    <div className="book-list">
       {books.map((book) => (
-        <li key={book.itemId || uuidv4()}>
-          <h3>{book.title}</h3>
-          <p>
-            Author:
-            {book.author}
-          </p>
-          <p>
-            Category:
-            {book.category}
-          </p>
-          <button type="submit" onClick={() => handleRemoveBook(book.itemId)}>Remove Book</button>
-        </li>
+        <div key={book.itemId || uuidv4()} className="book-details">
+          <div className="book-info">
+            <p className="School-of Text-Style-9">
+              {book.category}
+            </p>
+            <h3 className="Title Text-Style-5">{book.title}</h3>
+            <div className="author-buttons">
+              <p className="Suzanne-Collins Text-Style-8">
+                {book.author}
+              </p>
+              <ButtonGroup variant="text" aria-label="text button group">
+                <Button type="submit">Comments</Button>
+                <Button type="submit" onClick={() => handleRemoveBook(book.itemId)}>Remove</Button>
+                <Button type="submit">Edit</Button>
+              </ButtonGroup>
```

Comment:
- [ ] I think it would be great if you add a line between the comment remove and edit buttons to have the same ⭐😊 and please make it have the same color since it think you is darker than the given design 🙏🏻but please make it the same 


- Your
![image](https://github.com/Eccb7/book-store/assets/58553711/74fc0b1f-315b-4db3-b671-7a2c49b72d78)


- given
![image](https://github.com/Eccb7/book-store/assets/58553711/cb0d4f0b-3e33-44e7-ac7b-748a3df0baf0)